docs/installing.rst

```diff
@@ -0,0 +1,42 @@
+Installation
+============
+
+Required dependencies
+---------------------
+
+- Python 2.7, 3.4, 3.5, or 3.6
+- `numpy <http://www.numpy.org/>`__ (1.7 or later)
+- `cython <http://cython.org/>`__ (0.19 or later)
```

Comment:
I think Cython should only required for the development version? That was the case for netCDF4.

Reply:
You are correct. I'll fix that in a moment.
(Old habit, from before wheels, when people needed the build tool to pip install a package.)